.github/actions/update_submodules/action.yml

```diff
@@ -0,0 +1,9 @@
+name: "Update Submodules"
```

Comment:
We don't need this custom action because [`actions/checkout`](https://github.com/actions/checkout) has a `submodules` parameter.

Reply:
Hm, looks like this file is still around